.github/workflows/build-image.yaml

```diff
@@ -0,0 +1,33 @@
+name: Build AWS-Identity Image
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  copy-main-to-staging:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup crane
+        uses: imjasonh/setup-crane@00c9e93efa4e1138c9a7a5c594acd6c75a2fbf0c
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "1.19"
+      - run: docker build --no-cache -t ghcr.io/acorn-io/amazon-eks-pod-identity-webhook:${{github.ref}} .
+
+      - name: Copy Main to Staging
+        run: |
+          crane push ghcr.io/acorn-io/amazon-eks-pod-identity-webhook:${{github.ref}} ghcr.io/acorn-io/amazon-eks-pod-identity-webhook:${{github.ref}}
```

Comment:
should we push the tag and then retag to main? @StrongMonkey

Reply:
We don't really need to push to main. This can just go to per tag release.

Reply:
from the kustomize side then how do we know which tag to use for the deployment in helm?

Reply:
They are going to use the same tag. Once we verified that v0.4.0-acorn1 works for dev and staging, we will be promoting to production cluster.